crypto/txscript/test-data/script_tests-kip10.json

```diff
@@ -3591,28 +3592,21 @@
     "EVAL_FALSE"
   ],
   [
-    "2147483648 0",
```

Comment:
Can you keep test, and change result and description (and also keep the new one)?

Reply:
commit 955424cd4c1dd72f4c1028b3bcdd3dd017bb6309